AutomatticTracks/src/main/java/com/automattic/android/tracks/MessageBuilder.java

```diff
@@ -105,8 +105,8 @@ public static synchronized JSONObject createEventJSONObject(Event event, JSONObj
                 eventJSON.put(USER_TYPE_KEY, USER_TYPE_ANON);
                 eventJSON.put(USER_ID_KEY, event.getUser());
             } else {
+                eventJSON.put(USER_TYPE_KEY, event.getUserType().name().toLowerCase());
```

Comment:
I think that this code here `event.getUserType().name().toLowerCase()` may result in an invalid user type for WordPress.com users. In that case the string expected by the server is `wpcom:user:id`.
Should we change this code with a switch statements?

Did you add the usertype `simplenote` to the REST Endpoint?